modules/core/src/main/java/org/apache/ignite/IgniteSnapshot.java

```diff
@@ -48,4 +49,11 @@
      * @return Future which will be completed when cancel operation finished.
      */
     public IgniteFuture<Void> cancelSnapshot(String name);
+
+    /**
+     * Status snapshot operation.
```

Comment:
This is public API, please, extend the description.

Reply:
It's enough?